doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -262,7 +263,7 @@ This chapter describes the system parameters used by the components of the X-Roa
 | server-conf-client-cache-size                    | 100                                        |   |   | Maximum number of local clients to keep cached |
 | server-conf-service-cache-size                   | 1000                                       |   |   | Maximum number of services to keep cached |
 | server-conf-acl-cache-size                       | 100000                                     |   |   | Maximum number of access rights to keep cached in memory. |
-
+| enforce-certificate-validity-period-check        | false                                      |   |   | Whether to throw an exception about expired or not yet valid certificates. |
```

Comment:
Perhaps the parameter name should be ~ `enforce-client-is-certificate-validity-period` because there are also other certificates this setting does not control. The explanation could also be a bit more abstract, something like `whether to reject a request when client information system certificate is expired or not yet valid`; throwing and exception is an implementation detail.

Reply:
Changed the name